website/static/js/registrationUtils.js

```diff
@@ -198,6 +204,18 @@ var Question = function(data, id) {
     });
 
     /**
+     * @returns {Boolean} true if the value <input> does validate
+     **/
+    self.hasErrors = ko.computed(function() {
+        if( self.required ){
```

Comment:
Yes,
![screen shot 2015-08-10 at 11 58 50 am](https://cloud.githubusercontent.com/assets/8905795/9176091/371a6bfc-3f57-11e5-9f5c-74ae01b97cf8.png)

It is a function when the value has been extended